platform/kobo/disable-wifi.sh

```diff
@@ -9,9 +9,9 @@ ifconfig eth0 down
 
 # Some sleep in between may avoid system getting hung
 # (we test if a module is actually loaded to avoid unneeded sleeps)
-if lsmod | grep -q $WIFI_MODULE ; then
+if lsmod | grep -q "$WIFI_MODULE" ; then
```

Comment:
I can change it, but note that unquoted vars are basically always problematic whereas a lack of curly braces is only problematic in the case of `$array[key]` (should be `${array[key]}`). 

But more to the point, are you aware of a tool that would auto-check for that if we did that?